.github/workflows/twister.yaml

```
@@ -189,6 +189,7 @@ jobs:
           aws-region: us-east-2
 
       - name: ccache stats initial
+        if: always()
```

I do not think this does what it is supposed to do (i.e. let the subsequent steps execute if this step fails).

```suggestion
        continue-on-error: true
```

---

ok, but in this case we have to put this in the failing step that does the cache setup